src/main/java/com/cinema/shop/service/impl/AuthenticationServiceImpl.java

```diff
@@ -1,21 +1,27 @@
 package com.cinema.shop.service.impl;
 
 import com.cinema.shop.exceptions.AuthenticationException;
-import com.cinema.shop.lib.Inject;
-import com.cinema.shop.lib.Service;
 import com.cinema.shop.model.User;
 import com.cinema.shop.service.AuthenticationService;
 import com.cinema.shop.service.ShoppingCartService;
 import com.cinema.shop.service.UserService;
 import com.cinema.shop.util.HashUtil;
 import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
 
 @Service
 public class AuthenticationServiceImpl implements AuthenticationService {
-    @Inject
-    private UserService userService;
-    @Inject
-    private ShoppingCartService shoppingCartService;
+
+    private final UserService userService;
+    private final ShoppingCartService shoppingCartService;
+
+    @Autowired
```

Comment:
I think this annotation is redundant here :) and same in other classes

Reply:
just to be 100500% sure everything will work fine, but you-re right